Kerbee/Graph/GraphService.cs

```diff
@@ -118,6 +121,29 @@ public async Task RemoveManagedIdentityAsOwnerOfApplicationAsync(string applicat
             .DeleteAsync();
     }
 
+    public async Task RemoveCertificateAsync(string applicationObjectId, Guid keyId)
+    {
+        var client = GetClientForUser();
```

Comment:
Willen we dit uit de context van de gebruiker doen? Als we hier zijn is Kerbee nog eigenaar en kan zelf de certificates verwijderen. Misschien toch maar de ClientForManagedIdentity gebruiken?